web/docker/services/docker-compose.sqlite.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
```

Comment:
shouldn't be the version be 3.5 here too?

Reply:
No, we are using version `3.5` because of secrets. This file doesn't use secrets.